docs(devtools): add default environment variables (#1843)
scaleway/docs-content: developer-tools/devtools/reference-content.mdx

| `$SCW_INSECURE`                | Set this to `true` to enable the insecure mode                                                   | `$SCW_TLSVERIFY` (inverse flag used by the CLI)                                                               |
| `$SCW_PROFILE`                 | Set the configuration profile to use                                                                    | -                                                                                                             |
| `$SCW_PROJECT_ID`              | Your Project ID. ([Find out how to get your Project ID.](https://console.scaleway.com/project/settings))                     | `$SCW_DEFAULT_PROJECT_ID`                                                                                     |
| `$SCW_ORGANIZATION_ID` | Your Organization ID. ([Find out how to get your Organization ID.](https://console.scaleway.com/organization/settings))                   | `$SCW_DEFAULT_ORGANIZATION_ID`, `$SCW_ORGANIZATION` (used by the CLI),`$SCALEWAY_ORGANIZATION` (used by Terraform) |

> 

Reviewer: Could you add an example of configuration file also?

> Scaleway Devtools provide you several different ways to use and manage Scaleway products. You can find an extensive list of devtools documentation, sorted by product, to help you navigate the Scaleway ecosystem using Devtools.

Reviewer 2: link to https://www.scaleway.com/en/developers/ ?

Reviewer 2: Also I think this should be an index page and the content in a separate mdx ?

> ## Scalewat environment variables

Reviewer 2: ```suggestion
## Scaleway environment variables
```

| `$SCW_ACCESS_KEY`              | Access key of a token ([create yours](/identity-and-access-management/iam/how-to/create-api-keys/)) | `$SCALEWAY_ACCESS_KEY` (used by Terraform)                                                                 |
| `$SCW_SECRET_KEY`              | Secret key of a token ([create yours](/identity-and-access-management/iam/how-to/create-api-keys/)) | `$SCW_TOKEN` (used by the CLI), `$SCALEWAY_TOKEN` (used by Terraform), `$SCALEWAY_ACCESS_KEY` (used by Terraform) |                                                                                                         |
| `$SCW_DEFAULT_REGION`          | Your default [region](https://www.scaleway.com/en/developers/api/#regions-and-zones)             | `$SCW_REGION` (used by the CLI),`$SCALEWAY_REGION` (used by Terraform)                                        |
| `$SCW_DEFAULT_ZONE`            | Your default [availability zone](https://www.scaleway.com/en/developers/api/#regions-and-zones)  | `$SCW_ZONE` (used by the CLI),`$SCALEWAY_ZONE` (used by Terraform)                                            |

> | `$SCW_API_URL`                 | Url of the API                                                                                   | -                                                                                                             |

Author: ```suggestion
| `$SCW_API_URL`                 | URL of the API                                                                                   | -                                                                                                             |
```

> | `$SCW_INSECURE`                | Set this to `true` to enable the insecure mode                                                   | `$SCW_TLSVERIFY` (inverse flag used by the cli)                                                               |

Reviewer 3: ```suggestion
| `$SCW_INSECURE`                | Set this to `true` to enable the insecure mode                                                   | `$SCW_TLSVERIFY` (inverse flag used by the CLI)                                                               |
```

> | `$SCW_PROFILE`                 | Set the config profile to use                                                                    | -                                                                                                             |

Reviewer 4: ```suggestion
| `$SCW_PROFILE`                 | Set the configuration profile to use                                                                    | -                                                                                                             |
```

> | `$SCW_PROJECT_ID`              | Your project ID ([get yours](https://console.scaleway.com/project/settings))                     | `$SCW_DEFAULT_PROJECT_ID`                                                                                     |

Reviewer 3: ```suggestion
| `$SCW_PROJECT_ID`              | Your Project ID. ([Find out how to get your Project ID.](https://console.scaleway.com/project/settings))                     | `$SCW_DEFAULT_PROJECT_ID`                                                                                     |
```

> | `$SCW_ORGANIZATION_ID` | Your organization ID ([get yours](https://console.scaleway.com/organization/settings))                   | `$SCW_DEFAULT_ORGANIZATION_ID`, `$SCW_ORGANIZATION` (used by the CLI),`$SCALEWAY_ORGANIZATION` (used by Terraform) |

Reviewer 3: ```suggestion
| `$SCW_ORGANIZATION_ID` | Your Organization ID. ([Find out how to get your Organization ID.](https://console.scaleway.com/organization/settings))                   | `$SCW_DEFAULT_ORGANIZATION_ID`, `$SCW_ORGANIZATION` (used by the CLI),`$SCALEWAY_ORGANIZATION` (used by Terraform) |
```
